src/plugins/intel_myriad/myriad_plugin/myriad_plugin.cpp

```diff
@@ -170,7 +170,7 @@ QueryNetworkResult Engine::QueryNetwork(
             consoleOutput());
     std::set<std::string> namesToExclude;
     const auto supportedNetworks = vpu::FrontEnd::checkSupportedNetworks(network, namesToExclude);
-    for (const auto supportedNetwork : supportedNetworks) {
+    for (const auto& supportedNetwork : supportedNetworks) {
         const auto supportedLayers = getSupportedLayers(
```

Comment:
Add reference &, it will reduce copy constructor. It also fixs Ubuntu22.04 build fail issue.